File: benchmark-java/src/main/java/benchmark/java/output/IOutputHandler.java

package benchmark.java.output;

import benchmark.java.metrics.MetricResult;
import java.util.List;
import java.util.Map;


public interface IOutputHandler {
	

	public void handleBenchmarkResult(List<MetricResult> result);


	public void printBenchmarkInfo(Map<String, String> info);
}

File: benchmark-php/app/Output/IOutputHandler.php
<?php

namespace Benchmark\Output;


use Benchmark\Metrics\MetricResult;

interface IOutputHandler
{

	/**
	 * @param MetricResult[] $result
	 */
	public function handleBenchmarkResult($result);


	/**
	 * @param array $benchmarkInfo
	 */
	public function printBenchmarkInfo($benchmarkInfo);

}